hvantk/tests/ancestry/test_plot.py

```
@@ -0,0 +1,358 @@
+    ANCESTRY_PROB_COL,
+    SOURCE_COL,
+    KNOWN_ANCESTRY_COL,
+)
+
+
+@pytest.fixture
+def sample_predictions_df():
+    """Create sample predictions DataFrame for testing plots."""
+    np.random.seed(42)
+    n_query = 50
+    n_ref = 90
```

Ruff (0.14.14)

[error] Local variable `n_ref` is assigned to but never used

Remove assignment to unused variable `n_ref`

(F841)